File: android/src/main/java/com/baseflow/instancemanager/FlutterInstanceManagerPlugin.java
package com.baseflow.instancemanager;

import androidx.annotation.NonNull;
import io.flutter.embedding.engine.plugins.FlutterPlugin;

/**
 * Plugin class that exists because the Flutter tool expects such a class to exist for every Android
 * plugin.
 *
 * <p><strong>DO NOT USE THIS CLASS.</strong>
 */
public class FlutterInstanceManagerPlugin implements FlutterPlugin {
  @SuppressWarnings("deprecation")
  public static void registerWith(
      @NonNull io.flutter.plugin.common.PluginRegistry.Registrar registrar) {
    // no-op
  }

  @Override
  public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
    // no-op
  }

  @Override
  public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
    // no-op
  }
}

File: ios/Classes/FlutterInstanceManagerPlugin.m
#import "FlutterInstanceManagerPlugin.h"

// Plugin class that exists because the Flutter tool expects such a class to exist for every iOS
// plugin.
//
// <p><strong>DO NOT USE THIS CLASS.</strong>
@implementation FlutterInstanceManagerPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  // no-op
}

- (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
  // no-op
}

@end
